### PR TITLE
+pygmentize

### LINKED ISSUE #509: +pygments (42/548)
Formulae: https://formulae.brew.sh/formula/pygments

### PR DESCRIPTION
**Pull request by mfts**
closes #509

### CODE REVIEW

File: projects/pygments.org/package.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    
+  script: |
+    python-venv.sh {{prefix}}/bin/pygmentize
+
+provides:
+  - /bin/pygmentize
```
Comment (mxcl) [inline projects/pygments.org/package.yml]: the `/` prefix will evade our magic

Comment (mxcl) [reply projects/pygments.org/package.yml]: Certainly need some CI checks for this sort of thing

File: projects/pygments.org/package.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+versions:
+  github: pygments/pygments/tags
+
+dependencies:
+  python.org: 3.11
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
```
Comment (mxcl) [inline projects/pygments.org/package.yml]: doubt it needs either of these

File: projects/pygments.org/package.yml (hunk 1 above)
Comment (mfts) [reply projects/pygments.org/package.yml]: woah good catch. I don't know how that got that...

**Pull request merged**